rakefile.rb

```diff
@@ -17,13 +17,19 @@
 end
 
 desc "Compiles solution and runs unit tests"
-task :default => [:clean, :version, :compile, :xunit, :publish, :package]
+task :default => [:clean, :version, :nuget_restore, :compile, :xunit, :publish, :package]
 
 #Add the folders that should be cleaned as part of the clean task
 CLEAN.include(OUTPUT)
 CLEAN.include(FileList["src/**/#{CONFIGURATION}"])
 CLEAN.include(FileList["test/**/#{CONFIGURATION}"])
 
+desc "Restore NuGet packages"
+exec :nuget_restore do |cmd|
+   cmd.command = "tools/nuget/NuGet.exe"

```

Comment:
Nope, that does not exist. It has to be `dependencies/Nancy/tools/nuget/NuGet.exe`